mcpserver/args.go

```diff
@@ -0,0 +1,40 @@
+package mcpserver
+
+import "github.com/mark3labs/mcp-go/mcp"
+
+var (
+	explainationArgument = mcp.WithString("explanation",
```

Comment:
🎨 **Style (Low):** Typo in variable name - 'explanation' is misspelled as 'explaination'.

```suggestion
explanationArgument
```